tests/third_party/cupy/creation_tests/test_from_data.py

```diff
@@ -800,24 +804,27 @@ def test_cupy_array(self, dtype):
         assert a.shape == shape
 
 
+# TODO: ndmin=3 is removed since it needs ndarray.base
 @testing.parameterize(
     *testing.product(
-        {"ndmin": [0, 1, 2, 3], "copy": [True, False], "xp": [numpy, cupy]}
+        {"ndmin": [0, 1, 2], "copy": [True, False, None], "xp": [numpy, cupy]}
     )
 )
 class TestArrayCopy(unittest.TestCase):
-    @pytest.mark.skip("`ndmin` argument isn't supported")
     @testing.for_all_dtypes()
     def test_cupy_array(self, dtype):
+        if self.xp is numpy and self.copy is False:
+            pytest.skip()
+
         a = testing.shaped_arange((2, 3), self.xp, dtype)
         actual = cupy.array(a, copy=self.copy, ndmin=self.ndmin)
 
-        should_copy = (self.xp is numpy) or self.copy
+        should_copy = (self.xp is numpy) or (self.copy is True)
         # TODO(Kenta Oono): Better determination of copy.
         is_copied = not (
             (actual is a)
-            or (actual.base is a)
-            or (actual.base is a.base and a.base is not None)
+            #    or (actual.base is a)
+            #    or (actual.base is a.base and a.base is not None)
```

Comment:
it can be modified to handle `ndmin=3`
```suggestion
            or (self.xp is cupy)
            and (a.get_array()._pointer == actual.get_array()._pointer)
```